src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -263,6 +263,10 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.RulesetToEditorconfigConverter" Version="3.0.0" />
```

Comment:
I'd like to better understand this solution, let's sync up on this, as me asking tons of questions on the PR is not practical :)

Reply:
This is primarily used to convert the ruleset to .editorconfig.  Will explain in instructions